src/main/preferences.js

```diff
@@ -222,6 +225,24 @@ const preferencesConfig = {
 								options: [ { label: 'Lock the crosshair to the mouse cursor', value: 'followMouse' } ],
 								help: 'Keeps CrossOver centered on the mouse cursor. ',
 							},
+							{
+								label: 'Toggle Resize on ADS',
+								key: 'resizeOnADS',
+								type: 'radio',
+								options: [
+									{ label: 'Never', value: '0' },
```

Comment:
I think it may be more readable to use string values like `off`, `press`, `hold` rather than integers here, then it will be more obvious what's selected in iohook.js.

It's fine if you want to keep them as an int, just a suggestion.

Reply:
Yeah makes sense